services/slack/intermediate.go

```diff
@@ -127,6 +126,12 @@ func (t *Transformer) TransformUsers(users []SlackUser) {
 			Password:  model.NewId(),
 		}
```

Comment:
Something like this would be better
```suggestion
		newUser := &IntermediateUser{
			Id:        user.Id,
			Username:  user.Username,
			FirstName: user.Profile.FirstName,
			LastName:  user.Profile.LastName,
			Password:  model.NewId(),
		}
		
		if user.IsBot {
		newUser.Id = user.Profile.BotID
		}
		
```